WooCommerce/Classes/ViewRelated/Orders/Order Details/Issue Refunds/RefundConfirmationViewModel.swift

```diff
@@ -279,3 +287,44 @@ private extension RefundConfirmationViewModel {
         }
     }
 }
+
+private extension WCPayCardBrand {
+    /// A displayable brand name and last 4 digits for a card.
+    /// Names taken from [Stripe's card branding in the API docs](https://stripe.com/docs/api/cards/object#card_object-brand):
+    /// American Express, Diners Club, Discover, JCB, MasterCard, UnionPay, Visa, or Unknown.
+    func localizedCardDetails(last4: String) -> String {
+        return String(format: localizedCardDetailsFormatString(), last4)
+    }
+
+    func localizedCardDetailsFormatString() -> String {
+        switch self {
+        case .amex:
+            return NSLocalizedString("•••• %1$@ (American Express)",
+                                     comment: "American Express card brand details. %1$@ is a placeholder for the last 4 digits of the card.")
+        case .diners:
+            return NSLocalizedString("•••• %1$@ (Diners Club)",
+                                     comment: "Diners Club card brand details. %1$@ is a placeholder for the last 4 digits of the card.")
+        case .discover:
+            return NSLocalizedString("•••• %1$@ (Discover)",
+                                     comment: "Discover card brand details. %1$@ is a placeholder for the last 4 digits of the card.")
+        case .jcb:
+            return NSLocalizedString("•••• %1$@ (JCB)",
+                                     comment: "JCB card brand details. %1$@ is a placeholder for the last 4 digits of the card.")
+        case .mastercard:
+            return NSLocalizedString("•••• %1$@ (MasterCard)",
```

Comment:
Looking at [their website](https://brand.mastercard.com/brandcenter/branding-requirements/mastercard.html), it seems the right capitalization is `Mastercard` instead of `MasterCard`.

> When referencing Mastercard® in text, use an uppercase “M” and lowercase “c” with no space between “Master” and “card”. The name must not appear with a capital “C”. Mastercard must appear in the same font as its surrounding text. The name must not be modified in any way. The name may appear in all uppercase letters only if the font style of the user interface or communication also appears in all uppercase letters

Reply:
Thanks for checking and good catch. I've passed this info on to Stripe as it seems like the only one in that object which isn't correct for display.